packages/text-editor/src/components/CollaborativeTextEditor.svelte

```diff
@@ -202,6 +203,9 @@
     insertSeparatorLine: () => {
       editor?.commands.setHorizontalRule()
     },
+    insertEmoji: () => {
+      editor?.commands.setHorizontalRule()
```

Comment:
Why do you call `setHorizontalRule` here?

Reply:
Basically, it was built to showcase that the functionality can be extended to add emoji support so adding it as place holder for review purposes if required can be easily extended to support emoji popups. The screenshot shows that emoji command can be added to this doc in similar way

Reply:
Even though we implemented it in this way, I don't think that's the right way of extending this menu.
1. Currently this menu is supposed to insert only block elements. An emoji is an inline element and I don't see much value in inserting an emoji in the beginning of the line.
2. The way we use to extend the menu requires a lot of boilerplate. I believe we need to have a single list of commands that will be used in both bubble menu, inline menu and this side menu.

Reply:
On # 1 : Agree with you but many users are using it to start tech docs or quick start guides that's why added it in inline commands as well.

On # 2 : I followed the existing implementation flow as it was done and tried to extend this support but based upon your comment looks like there might be a need to implement it differently let me know if I can try that way.